modules/svg/include/SkSVGApi.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SkSVGAPI_DEFINED
+#define SkSVGAPI_DEFINED
+
+#include <string>
+#include <vector>
+
+class __attribute__((visibility("default"))) SkSVGApi {
+ public:
+    static bool registerFonts(const std::vector<std::string>& fontFilePaths);
+    static bool registerFonts(const std::vector<std::pair<std::string, std::string>>& fontFilePaths);
```

Comment:
One API to register fonts, where the font family is determined internally. The other is a pair of font paths and font families, so we can override it however we want to match SVGs